### ISSUE TITLE
Generated `BUCK` file points to a URL that does not exist

### ISSUE THREAD

**Issue by steveklabnik** [facebook/buck2#139]
When you generate a new project with `buck2 init --git`, the generated `BUCK` file contains this line:

````
# A list of available rules and their signatures can be found here: https://buck2.build/docs/generated/starlark/prelude/prelude.bzl
````

However, this URL does not exist.

**Comment by ndmitchell** [comment]
https://buck2.build/docs/api/rules/ is the right URL now (I changed it to be shorter and easier to read) - I'll go fix.

**Comment by get9** [comment]
Fixed in c77ff3f4eb10124b1fd8cc128327ca807ed9b741
